src/hoc/Aux/Aux.js

```diff
@@ -0,0 +1,3 @@
+const aux = (props) => props.children;
```

Comment:
aux function is not a higher-order component. It's just a utility function.

Reply:
Understood 👍🏽 
That was redundant also, it'll be removed in next push.